exec/cpg.c

```diff
@@ -724,30 +733,37 @@ static int notify_lib_joinlist(
 		}
 	}
 
+	/*
+	 * Fill res->left_list
+	 */
 	if (left_list_entries) {
 		memcpy (retgi, left_list, left_list_entries * sizeof(mar_cpg_address_t));
 		retgi += left_list_entries;
 	}
 
 	if (joined_list_entries) {
-		int i;
-
 		memcpy (retgi, joined_list, joined_list_entries * sizeof(mar_cpg_address_t));
 		retgi += joined_list_entries;
 
```

Comment:
the other two instances of filling res->XXX_list now have comments, so this might get one as well (although it's pretty obvious waht happens..)

Reply:
Yep, agree